src/components/DatBoolean.vue

```diff
@@ -1,23 +1,38 @@
 <script setup lang="ts">
+import { computed } from "vue";
+
 interface Props {
   label?: string;
+  disabled?: boolean;
 }
 
-withDefaults(defineProps<Props>(), {
+const props = withDefaults(defineProps<Props>(), {
   label: "",
+  disabled: false,
+});
+const watchDisabled = computed(() => {
+  return props.disabled;
 });
```

Comment:
Could you apply this suggestion? (in this file and all others too)
I think I find it easier to read. (don't forget to rename `watchDisabled` to `isDisabled` (in this file and all others too)
Thanks! :pray: 

```suggestion
const isDisabled = computed(() => props.disabled);
```